sdk/parents/azure-sdk-parent/pom.xml

```diff
@@ -283,6 +283,22 @@
           <artifactId>maven-install-plugin</artifactId>
           <version>3.1.3</version> <!-- {x-version-update;org.apache.maven.plugins:maven-install-plugin;external_dependency} -->
         </plugin>
+
+        <plugin>
```

Comment:
How can I tell in a CI pipeline that this actually ran? Looking at the logs from the CI jobs ran for this PR and those from previous runs they don't appear to be taking any longer nor are there logs mentioning that OWASP is running.

Reply:
@alzimmermsft It did not run as part of CI. I need to speak with Eng Sys to see if we can persist the database. Otherwise, this would increase pipelines by 30+ mins which is not acceptable.

Reply:
Same reply as my other comment